docs/formats.md

```diff
@@ -203,20 +238,254 @@ Creates a ES6 module of the style dictionary.
+
+
+Creates a dimen resource xml file with all the font sizes in your style dictionary.
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+ <dimen name="size_font_tiny">10.00sp</dimen>
+ <dimen name="size_font_small">13.00sp</dimen>
+ <dimen name="size_font_medium">15.00sp</dimen>
+```
+
+* * *
+
+### android/integers 
+
+
+Creates a resource xml file with all the integers in your style dictionary. It filters your
+style properties by `prop.attributes.category === 'time'`
+
+**Todo**
+
+- Update the filter on this.
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <integer name="time_duration_short">1000</string>
+  <integer name="time_duration_medium">2000</string>
+  <integer name="time_duration_long">4000</string>
+```
+
+* * *
+
+### android/strings 
+
+
+Creates a resource xml file with all the strings in your style dictionary. Filters your
+style properties by `prop.attributes.category === 'content'`
+
+**Example**  
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <string name="content_icon_email">&#xE001;</string>
+  <string name="content_icon_chevron_down">&#xE002;</string>
+  <string name="content_icon_chevron_up">&#xE003;</string>
+```
+
+* * *
+
+### ios/macros 
+
+
+Creates an Objective-C header file with macros for style properties
+
+**Example**  
+```objectivec
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#define ColorFontLink [UIColor colorWithRed:0.00f green:0.47f blue:0.80f alpha:1.00f]
+#define SizeFontTiny 176.00f
+```
+
+* * *
+
+### ios/plist 
+
+
+Creates an Objective-C plist file
+
+**Todo**
+
+- Fix this template and add example and usage
+
+
+* * *
+
+### ios/singleton.m 
+
+
+Creates an Objective-C implementation file of a style dictionary singleton class
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/singleton.h 
+
+
+Creates an Objective-C header file of a style dictionary singleton class
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/static.h 
+
+
+Creates an Objective-C header file of a static style dictionary class
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/static.m 
+
+
+Creates an Objective-C implementation file of a static style dictionary class
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/colors.h 
+
+
+Creates an Objective-C header file of a color class
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/colors.m 
+
+
+Creates an Objective-C implementation file of a color class
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/strings.h 
+
+
+Creates an Objective-C header file of strings
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### ios/strings.m 
+
+
+Creates an Objective-C implementation file of strings
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### css/fonts.css 
+
+
+Creates CSS file with @font-face declarations
+
+**Todo**
+
+- Add example and usage
+
+
+* * *
+
+### static-style-guide/index.html 
```

Comment:
in the future, I would like to fix this: it's not clear what it is, how to use it, etc.

Reply:
I submitted a different PR to remove it entirely. I was naive when I created it and it has only generated confusion.

Reply:
🙌